src/main/java/mods/railcraft/api/core/items/ISafetyPants.java

```diff
@@ -0,0 +1,19 @@
+package mods.railcraft.api.core.items;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+
+public interface ISafetyPants {
```

Comment:
Please add as little bit of Javadoc explaining when it is appropriate to use this interface.